MapboxNavigation/NavigationMapView.swift

```diff
@@ -1350,3 +1380,128 @@ extension NavigationMapView {
     }
 }
 
+// MARK: - Building Extrusion Highlights
+
+private struct BuildingHighlightAttributes: Hashable {
+    static let kDefaultIdentifier: Int64 = -1
+    
+    var coordinate: CLLocationCoordinate2D = kCLLocationCoordinate2DInvalid
+    var highlightColor: UIColor = .defaultBuildingHighlightColor
+    var identifier: Int64 = kDefaultIdentifier
+    
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(identifier)
+    }
+    
+    static func ==(lhs: BuildingHighlightAttributes, rhs: BuildingHighlightAttributes) -> Bool {
+        return lhs.identifier == rhs.identifier
+    }
+}
+
+extension NavigationMapView {
+       
+    /**
+     Receives coordinates for searching the map for buildings. If buildings are found, they will be highlighted in 2d or 3d depending on the `in3D` value.
+     `in3D` defaults to true
+     `buildingHighlightingEnabled` must be set to true before this function will work.
+     */
+    public func highlightBuildings(for coordinates: [CLLocationCoordinate2D], in3D: Bool = true) {
+        let buildingAttributes = coordinates
+            // Create building attributes by searching for building ID by provided coordinate.
+            .map({ BuildingHighlightAttributes(coordinate: $0,
+                                               highlightColor: buildingHighlightColor,
```

Comment:
I think best idea is to keep current initial implementation as simple as possible. I think it's even better to use `Set<Int64>` instead of `Dictionary<Int64, CLLocationCoordinate2D>` since `CLLocationCoordinate2D` is only used to get building identifier.